readme.md

```diff
@@ -6,16 +6,18 @@ Firmata is a protocol for communicating with microcontrollers from software on a
 
 # Contents
 
-- [Usage](#usage)
-- [Firmata Client Libraries](#firmata-client-libraries)
-- [Updating Firmata in the Arduino IDE - Arduino 1.6.4 and higher](#updating-firmata-in-the-arduino-ide---arduino-164-and-higher)
-- [Cloning Firmata](#cloning-firmata)
-- [Updating Firmata in the Arduino IDE - older versions (<= 1.6.3 or 1.0.x)](#updating-firmata-in-the-arduino-ide---older-versions--163-or-10x)
-  - [Mac OSX:](#mac-osx)
-  - [Windows](#windows)
-  - [Linux](#linux)
-- [Using the Source code rather than release archive (only for versions older than Arduino 1.6.3)](#using-the-source-code-rather-than-release-archive-only-for-versions-older-than-arduino-163)
-- [Contributing](#contributing)
+- [Firmata](#firmata)
```

Comment:
Is this block of changes intentional?

Reply:
Oops no, sorry. This changes made by vscode editor itself automatically to add all sections to contents. Feel free to close or merge this. I can open new PR without this changes.

Reply:
Please update to remove those changes.